cves/kernel/CVE-2021-20265.yml

```diff
@@ -75,7 +78,8 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: 
+  - https://bugzilla.redhat.com/show_bug.cgi?id=1908827,
```

Comment:
I think you want this to read either:

bugs:  https://bugzilla.redhat.com/show_bug.cgi?id=1908827

or

bugs: | 
  https://bugzilla.redhat.com/show_bug.cgi?id=1908827

I don't think yml is checking the new line without the vertical bar